tests/baselines/reference/assignmentCompatWithDiscriminatedUnion.errors.txt

```diff
@@ -6,10 +6,6 @@ tests/cases/conformance/types/typeRelationships/assignmentCompatibility/assignme
 tests/cases/conformance/types/typeRelationships/assignmentCompatibility/assignmentCompatWithDiscriminatedUnion.ts(58,5): error TS2322: Type 'S' is not assignable to type 'T'.
   Property 'c' is missing in type 'S' but required in type '{ a: 2; b: 4 | 3; c: string; }'.
 tests/cases/conformance/types/typeRelationships/assignmentCompatibility/assignmentCompatWithDiscriminatedUnion.ts(82,5): error TS2322: Type 'S' is not assignable to type 'T'.
-  Type 'S' is not assignable to type '{ a: N; b: N; c: 2; }'.
-    Types of property 'c' are incompatible.
-      Type 'N' is not assignable to type '2'.
-        Type '0' is not assignable to type '2'.

```

Comment:
I'm not a fan of these errors not being included anymore and I'll try to keep them, but IMO the diagnostic changes in the other files do make sense.

Reply:
@DanielRosenwasser @ahejlsberg Wait I actually think that those errors are a bug (?). The example says that `s` should be assignable to `t` (and if you look at it, it is), but it errors because of the number of combinations being [too complex](https://github.com/microsoft/TypeScript/blob/68d8be4f1d89d94a543d93fcedad17f3e4ef297b/tests/cases/conformance/types/typeRelationships/assignmentCompatibility/assignmentCompatWithDiscriminatedUnion.ts#L63-L82). The diagnostics are incorrect, and so I'm actually prefer them going away.